presentation-api/index.html

```diff
@@ -0,0 +1,111 @@
+---
+feature_name: Presentation API
+chrome_version: 51
+feature_id: 6676265876586496
+---
+
+<h3>Background</h3>
+
+<p>
+  This sample illustrates the use of
+  <a href="https://w3c.github.io/presentation-api/">Presentation API</a>,
+  which gives the ability to access external presentation-type displays and use
+  them for presenting web content. The <code>PresentationRequest</code> object
+  is associated with a request to initiate or reconnect to a presentation made
+  by a controlling browsing context and it takes in a presentation request URL
+  when constructed. A presentation can be started by calling the
+  <code>start()</code> method on the <code>PresentationRequest</code> object.
+</p>
+
+{% capture initial_output_content %}
+<p>
```

Comment:
I used a `<p>` tag so that there will be some margin between the form fields without the need for extra css markup. 

![00pqy6zpp8b](https://cloud.githubusercontent.com/assets/2845328/16051770/fd2ddd86-3215-11e6-8b95-dc364260998d.png)
![veamyeafndh](https://cloud.githubusercontent.com/assets/2845328/16051769/fd2c5ec0-3215-11e6-9a52-768563d23958.png)


w3c also [considers each part of a form a paragraph](https://www.w3.org/TR/html5/forms.html#writing-a-form's-user-interface):

> Each part of a form is considered a paragraph, and is typically separated from other parts using p elements.